src/cmd/initialize.go

```diff
@@ -220,5 +220,7 @@ func init() {
 
```

Comment:
You'll have to run `make docs-and-schema` to update the CLI docs with the new key flag.